### PR TITLE
Added multiple components to front-end, worked on css and deleted unnecessary files.

### PR DESCRIPTION
**Pull request by tonaramos**
New pull request with additions after deleting previous pulled request that contained conflicts.

### CODE REVIEW

File: aDataGenerator/dataGenerator.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+const randomInteger = (min, max) => Math.floor(Math.random() * ((max - min) + 1)) + min;
+
+const arbitrarySentence = (arrEntry, numOfWords, hasPeriod) => {
+  let outputString = '';
+  for (let i = 0; i < numOfWords; i += 1) {
+    if (i === 0) {
+      let word = arrEntry[randomInteger(0, arrEntry.length - 1)].split('');
+      word[0] = word[0].toUpperCase();
+      word = word.join('');
```
Comment (clfhhc) [inline aDataGenerator/dataGenerator.js:88]: I think there's no need to split and join to make the first letter capital

File: aDataGenerator/dataGenerator.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+  for (let i = 0; i < randomInteger(0, 3); i += 1) {
+    const obj = {
+      startDate: new Date(2018, 7 + count, randomInteger(1, 15)),
+      endDate: new Date(2018, 7 + count + 1, randomInteger(16, 30)),
+      nightsOfMinimumStay: randomInteger(4, 6),
+    };
+    output.push(obj);
+    count += 1;
+    count += randomInteger(1, 2);
```
Comment (clfhhc) [inline aDataGenerator/dataGenerator.js:143]: why not combine them?

File: client/src/js/components/HomeDescription.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  hideDescription() {
+    this.setState({ displayDescription: false });
+  }
+
+  showDescription() {
+    this.setState({ displayDescription: true });
+  }
+
+  render() {
+    const descriptionContent = this.props.data.map((item, i) =>
```
Comment (clfhhc) [inline client/src/js/components/HomeDescription.jsx:22]: Did you miss a parenthesis at the end of line?

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const db = require('../database/index');
+const bodyParser = require('body-parser');
+const homeRoutes = require('./homes');
```
Comment (clfhhc) [inline server/index.js:4]: Did we agree to use /room as the route?

**Review by clfhhc — APPROVED** [review]
Looks good

File: aDataGenerator/dataGenerator.js (hunk 1 above)
Comment (tonaramos) [reply aDataGenerator/dataGenerator.js:88]: I believe toUpperCase did not work in the first character of the string so I had to split it. I'll look into it though

File: aDataGenerator/dataGenerator.js (hunk 2 above)
Comment (tonaramos) [reply aDataGenerator/dataGenerator.js:143]: good call

File: server/index.js (hunk 4 above)
Comment (tonaramos) [reply server/index.js:4]: oh, might have to update that.

**Pull request merged**